libs/core/src/components/pds-table/pds-table-head/test/pds-table-head.spec.tsx

```diff
@@ -0,0 +1,78 @@
+import { newSpecPage } from '@stencil/core/testing';
+import { PdsTableHead } from '../pds-table-head';
+import { PdsTable } from '../../pds-table';
+
+describe('pds-table-head', () => {
+  it('renders', async () => {
+    const page = await newSpecPage({
+      components: [PdsTableHead],
+      html: `<pds-table-head></pds-table-head>`,
+    });
+    expect(page.root).toEqualHtml(`
+      <pds-table-head role="row">
+        <mock:shadow-root>
+          <slot></slot>
+        </mock:shadow-root>
+      </pds-table-head>
+    `);
+  });
+
+  it('renders with is-fixed class when tableRef has fixedColumn', async () => {
+    const page = await newSpecPage({
+      components: [PdsTableHead, PdsTable],
+      html: `
+        <pds-table fixed-column="true">
+          <pds-table-head>
```

Comment:
Same comment regarding the lack of `pds-table-body`.

Reply:
This is a test for the table head, not sure why I need to include the body?

Reply:
same as other table-head comments